src/wasm/wasm-validator.cpp

```diff
@@ -2475,45 +2470,47 @@ void FunctionValidator::visitTryTable(TryTable* curr) {
                 curr,
                 "the number of catch tags and sent types do not match");
 
+  const char* invalidSentTypeMsg = "invalid catch sent type information";
   Type exnref = Type(HeapType::exn, Nullable);
   for (Index i = 0; i < curr->catchTags.size(); i++) {
-    Name tagName = curr->catchTags[i];
-    if (!tagName) {
-      continue;
-    }
+    auto sentType = curr->sentTypes[i];
+    size_t tagTypeSize;
 
-    // Check tag validity
-    auto* tag = getModule()->getTagOrNull(tagName);
-    if (!shouldBeTrue(tag != nullptr, curr, "")) {
-      getStream() << "catch's tag name is invalid: " << tagName << "\n";
-    } else if (!shouldBeEqual(tag->sig.results, Type(Type::none), curr, "")) {
-      getStream()
-        << "catch's tag (" << tagName
-        << ") has result values, which is not allowed for exception handling";
-    }
+    Name tagName = curr->catchTags[i];
+    if (!tagName) { // catch_all or catch_all_ref
+      tagTypeSize = 0;
+
```

Comment:
nit: I think this extra newline looks a little odd.

Reply:
Removed it